### DISCUSSION TITLE
"pdxinfo file not found" error when attempting to launch game in simulator.

### DISCUSSION THREAD

**Discussion by bittoh** [NobleRobot/NobleEngine#59] [Support Requests]
**Describe the bug**
When building the game using the VSCode Playdate Template, the game builds successfully, but fails to load with a "pdxinfo file not found" error, even though the pdxinfo file is present in the correct directory ("source" folder).

**To reproduce**
Combine NobleEngine template project with https://github.com/Whitebrim/VSCode-PlaydateTemplate .
Use the Run build task... function in VSCode to build a pdx file and run it in the simulator.

**Expected behavior**
The pdxinfo file is present in the source folder.
The game should boot normally without any errors in the simulator.

**Screenshots/Code snippets**
![image](https://github.com/NobleRobot/NobleEngine/assets/114200345/f38da972-21c7-4641-9121-d89078d57a2a)

**Comment by Mark-LaCroix** [comment]
I know that that the creator of that template uses Noble Engine so I can't explain why this isn't working, but I haven't tested Noble Engine with that template and I can't offer official support for it. This is probably better addressed in the issue/discussion section of that project.

The problem could be with that project's build script, or it _might_ happen if you forgot to put the engine code in the `libraries` folder of the Noble Engine template project (either as a git submodule or by downloading it manually).

Based only on the error message, it seems like the build script is looking for `pdxinfo` in the wrong folder. Again, this is an issue with that project (perhaps something you introduced when you merged it with the Noble Engine project template), not this one.
